packages/ui/src/components/pagination.tsx

```
@@ -1,117 +1,127 @@
-  <nav
-    role="navigation"
-    aria-label="pagination"
-    className={cn("mx-auto flex w-full justify-center", className)}
-    {...props}
-  />
-);
-Pagination.displayName = "Pagination";
+function Pagination({ className, ...props }: React.ComponentProps<"nav">) {
+  return (
+    <nav
+      role="navigation"
```

Biome (1.9.4)

[error] Using the role attribute 'navigation' on the 'nav' element is redundant, because it is implied by the semantic 'nav' element.

Unsafe fix: Remove the role attribute.


(lint/a11y/noRedundantRoles)